package.toml.template

```diff
@@ -0,0 +1,5 @@
+[buildpack]
+uri = "./"
+
+[platform]
+os = "linux"
```

Comment:
What's this file for?

Reply:
removed, this was to help build the package locally but was no longer required